monitoring/jaeger-mixin/README.md

```diff
@@ -0,0 +1,80 @@
+# Prometheus monitoring mixin for Jaeger
+
+The Prometheus monitoring mixin for Jaeger provides a starting point for people wanting to monitor Jaeger using Prometheus, Alertmanager, and Grafana. To use it, you'll need [`jsonnet`](https://github.com/google/go-jsonnet) and [`jb` (jsonnet-bundler)](https://github.com/jsonnet-bundler/jsonnet-bundler). They can be installed using `go get`, as follows:
+
```

Comment:
It says "using Prometheus, Alertmanager and Grafana" - but doesn't make it clear what are the pre-requisites that must be installed before this would work? For example - ServiceMonitor and PodMonitor are mentioned - but assume this requires the Prometheus Operator?

The `kubectl apply -f manifests/` further down - what does this require to be installed to work? It is adding configuration for Grafana? is there a grafana operator that needs to be installed to watch for those manifests?

Reply:
No pre-requisites, everything that is needed is generated in the `manifest` directory. Among the manifests is the Prometheus Operator, which handles the Grafana configuration as well (AFAIK).

Reply:
Ok, might be worth adding something about that in the text?

Reply:
How about this? 

> The resulting manifests will include everything that is needed to have a Prometheus, Alertmanager, and Grafana instances. Whenever a new alert rule is needed, or a new dashboard has to be defined, change your `monitoring-setup.jsonnet`, re-generate and re-apply the manifests.

Reply:
Looks good.